src/common/hooks/use-fetch.ts

```diff
@@ -0,0 +1,9 @@
+const hiroHeaders: HeadersInit = {
+  'x-hiro-product': 'heystack',
+  'x-hiro-version': '1.0.0', // TODO: Global VERSION
```

Comment:
Can pull in direct from package.json?

Reply:
I ported this over from the web wallet, but maybe don't need.